cocotb/drivers/amba.py

```diff
@@ -78,12 +78,8 @@ def _send_write_address(self, address, delay=0):
         self.bus.AWADDR <= address
         self.bus.AWVALID <= 1
 
-        while True:
-            yield ReadOnly()
-            if self.bus.AWREADY.value:
-                break
-            yield RisingEdge(self.clock)
-        yield RisingEdge(self.clock)
+        yield StableValue(self.bus.AWREADY, 1, event=RisingEdge(self.clock),
+                          extra_event=True)
```

Comment:
For the sake of argument, do other reviewers think this is clearer than the following?
```python
yield StableCondition(lambda: self.bus.AWREADY == 1, event=RisingEdge(self.clock),
                      extra_event=True)
```

Reply:
Reason I kept ``StableValue`` is that seems to be the common case.